src/components/main-form.js

```diff
@@ -50,33 +50,20 @@ option2.term = 30
 
 // CSS
 
-const Summary = styled(Row)`
+// TODO 2019-09-15: Where should global styles live?
+injectGlobal`
   b {
     font-weight: 500;
   }
+`
 
-  p,
-  h5 {
-    font-size: 17px;
-  }
-
-  h5 {
-    margin-bottom: 1rem;
-    margin-left: 1.5rem;
-  }
-
-  .col {
-    margin-left: -2rem;
-  }
-
-  ol {
-    font-size: 15px;
-    margin-right: 2.5rem;
-  }
+// TODO 2019-09-15: Should these be global?
+const Ol = styled('ol')`
```

Comment:
I think we can just leave it as is for now and we can make it global in the future if we need to